core/runtime/src/main/java/io/quarkiverse/jberet/runtime/QuarkusJobExecutor.java

```diff
@@ -1,19 +1,22 @@
 package io.quarkiverse.jberet.runtime;
 
-import java.util.concurrent.Executor;
-
+import io.quarkus.runtime.ThreadPoolConfig;
 import org.jberet.spi.JobExecutor;
+import org.wildfly.common.cpu.ProcessorInfo;
+
+import java.util.concurrent.Executor;
 
 class QuarkusJobExecutor extends JobExecutor {
-    public QuarkusJobExecutor(Executor delegate) {
+    private final ThreadPoolConfig threadPoolConfig;
```

Comment:
If we do the calculation of the thread pool in the constructor, we don't need this as a field.

Reply:
agreed and changes are done